net-dns/dnsdist/metadata.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>bgo@9dt.de</email>
+		<name>Vladimir Datsevich</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<longdescription lang="en">
+	</longdescription>
+	<use>
+		<flag name="clang">Use Clang compiler instead of GCC</flag>
```

Comment:
That's not a valid case for a use flag. If people want to use clang, they can set CC&CXX.